README.markdown

```diff
@@ -189,7 +189,7 @@ github "PromiseKit/MapKit" ~> 1.0
 
 ## Choose Your Networking Library
 
-`NSURLSession` is typically inadequate; choose from [Alamofire] or [OMGHTTPURLRQ]:
+`URLSession` is typically inadequate; choose from [Alamofire] or [OMGHTTPURLRQ]:
```

Comment:
Since the sample code below is in Swift I don't find it strange to prefer `URLSession` here.